preview/MsixCore/msixmgr/FilePaths.cpp

```diff
@@ -126,6 +129,8 @@ HRESULT FilePathMappings::InitializePaths()
     m_map[L"AppVSystems32Driverstore"] = appVSystem32DriverstorePath;
     m_map[L"AppVSystems32Logfiles"] = appVSystem32LogfilesPath;
     m_map[L"AppVSystems32Spool"] = appVSystem32SpoolPath;
+    m_map[L"Local AppData"] = std::wstring(localAppDataPath.Get());
```

Comment:
Shouldn't it say `LocalAppData` instead?

Reply:
The AppV VFS path for LocalAppData actually has a space in it.